src/server/game.js

```diff
@@ -357,30 +464,19 @@ let game = function() {
 				this.spawnMarble(id, name, attributes);
 
 				// Wait for a human entering the round before starting it
-				if (_isWaitingForEntry) {
-					_isWaitingForEntry = false;
-					this.setCurrentGameState(gameConstants.STATE_ENTER);
-
-					// Start the game after the entering period is over
-					clearTimeout(this.enterTimeout);
-					this.enterTimeout = _setTrackableTimeout(
-						this.start.bind(this),
-						config.marbles.rules.enterPeriod * 1000
-					);
-				}
+				if(_currentGameState === gameConstants.STATE_WAITING)
+					_setCurrentGameState(gameConstants.STATE_ENTER);
 			}
 		},
 
 		// Spawns marble unless the maximum amount of marbles has been hit
 		spawnMarble(id, name, attributes) {
-			if (
-				// Check whether the game state disallows new marbles
-				this.currentGameState === gameConstants.STATE_FINISHED
-				// Check whether we have reached the maximum marble limit
-				|| _marbles.length >= config.marbles.rules.maxMarbleCount) return;
+			if (_currentGameState === gameConstants.STATE_FINISHED || _currentGameState === gameConstants.STATE_LOADING || _marbleLimitReached)
+				return;
 
 			// Start physics simulation if this is the first marble
-			if(_marbles.length === 0) physics.world.startUpdateInterval();
+			if(_marbles.length === 0)
+				physics.world.startUpdateInterval();
```

Comment:
I can't remember why we actually have the single-line rule, but I'll fix it either way!

Reply:
I think the main reason is that a second line without brackets could be confusing. The only difference that makes it visible that it's a conditional statement is the indent. Having it on a single line helps, considering you read left to right.
However, nowadays I'm tempted to say to change the code style to always include brackets. It might become overly verbose in some locations but overall it should help with readability.
Then again, why bother to keep changing things, might as well just stick with what we have, it works fine.